icubam/www/templates/disclaimer.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block links %}
+{% end %}
+
+{% block content %}
+<div class="modal-body">
+  <p>{% raw disclaimer %}</p>
```

Comment:
Maybe the `<p>` should be in the disclaimer HTML? Otherwise it can't contain several paragraphs.

Reply:
Remove <p> tag on 9e9e253